### PR TITLE
Feature/new pa endpoints

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: src/pages/search/utils/transformData.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+export const transformPAValues = (rawData, totalArea) => {
```
Comment (erikasv) [inline src/pages/search/utils/transformData.js:1]: Maybe we could add here the function `processData` that is currently in Ecosystems component ? Or do you think is out of scope for this file?

**Review by erikasv — COMMENTED** [review]
Great job organizing EcosystemDetails :tada:

File: src/pages/search/utils/transformData.js (hunk 1 above)
Comment (cazapatamar) [reply src/pages/search/utils/transformData.js:1]: Yes, agree. The idea is to use this file to centralize the functions that transform data to be passed to graph components specially those that are used from diferent components.

Comment (cazapatamar) [reply src/pages/search/utils/transformData.js:1]: Done!

**Comment by cazapatamar** [comment]
When there is not data of PA for an speciic SE, a message `Sin áreas protegdas` is shown. It is already validated with Cesar.

**Pull request merged**